src/runtime_src/core/common/api/xrt_ip.cpp

```diff
@@ -111,6 +111,13 @@ class ip::interrupt_impl
     device->wait_ip_interrupt(handle);
     enable(); // re-enable interrupts
   }
+
+  std::cv_status
+  wait(const std::chrono::duration<int32_t, std::milli>& timeout) const
```

Comment:
Let's use the helper type `std::chrono::milliseconds` here are everywhere

Reply:
done